CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Cromwell's metadata archival configuration has changed in a backwards incompatib
 please see
 [the updated documentation](https://cromwell.readthedocs.io/en/stable/Configuring#hybrid-metadata-storage-classic-carbonite) for details.
 
+### Bugfixes
+
++ Fixed a bug where the `use_relative_output_paths` option would not preserve intermediate folders.
```

Comment:
Unfortunately this comment must now be relocated to the 51 release notes.

We apologize you didn't make the release, was not on our radar.

Reply:
I have updated the release note.

No need to apoligize. The cromwell team is doing so much already. It's a lot of work to maintain a worklow engine. I don't take that for granted :slightly_smiling_face: . 
Would it help if I attach the "on deck for review" label myself next time I make a small bug or documentation fix? These should be pretty low effort to review and merge.

Reply:
We have a weekly meeting where we select the highest priority PRs and assign them "on deck for review", so self-assignment would not be compatible with that process 😄

Reply:
I will attach the HIGH PRIORITY label then :wink:. But only for small bugfixes like this. I promise I won't be annoying :innocent: .